src/components/title-page-components/SequentialOrder.tsx

```diff
@@ -47,6 +47,8 @@ const SequentialAdjacent = styled(SequentialItem)<{ $alignRight?: boolean }>`
 				margin-right: auto;
 			`};
 
+	height: 100%;
+
 	img {
 		max-height: 50%;
 	}
```

Comment:
I think to make the posters behave more consistently, I would recommend only every setting the width, never the max height.

In this case, perhaps something like `width: 10rem`.